frontend/css/music-services.less

```diff
@@ -1,5 +1,18 @@
 @checkbox-size: 40px;
 
+#user-profile {
+  .card {
+    margin-bottom: 2rem;
+    .card-header {
+      padding: 1rem 1.5rem;
+      .card-title {
+        font-size: 16px;
+        margin: 0;
+      }
```

Comment:
Would be worth investigating if this needs to be changed for the whole website or only applies to this component